lib/rules/no-extra-parens.js

```diff
@@ -428,6 +429,38 @@ module.exports = {
             }
         }
 
+        /**
+         * Check the parentheses around the super class of the given class definition.
+         * @param {ASTNode} node The node of class declarations to check.
+         * @returns {void}
+         */
+        function dryClass(node) {
```

Comment:
Sorry, I didn't realize the other methods were also prefixed with `dry`. In that case, I won't block this PR just because of that, we can rename the methods in a separate PR.